README.md

```diff
@@ -62,6 +62,8 @@ In this package, right now we have the following extension methods:
 - **Extensions on num:**
     - SizedBox hSizedBox
     - SizedBox wSizedBox
+- **Extension on Iterable**
```

Comment:
Add a <br> before this.

Reply:
I did not understand this.

Reply:
Oh haha! You need to add a `<br>` tag before this line.

Reply:
@AbhishekDoshi26 Done